src/Material.cc

```diff
@@ -358,3 +366,64 @@ void Material::SetFilePath(const std::string &_filePath)
 {
   this->dataPtr->filePath = _filePath;
 }
+
+//////////////////////////////////////////////////
+ignition::math::Color Material::Implementation::GetColor(
+    const std::string &_colorType, Errors &_errors)
+{
+  ignition::math::Color color(0, 0, 0, 1);
+
+  if (!this->sdf->HasElement(_colorType))
+    return color;
```

Comment:
Before we return the default color here, should we either print some sort of warning message that lets the user know they're getting the default color (and why), or append this warning message to `_errors`?

Reply:
Since this is the expected behavior, we shouldn't print a warning for this. I've verified with @azeey offline